src/sxastarter/src/components/LayoutContainer.tsx

```diff
@@ -45,12 +63,19 @@ export const Default = (props: ComponentProps): JSX.Element => {
 
   const id = props.params.RenderingIdentifier;
 
+  const stackColumnAt = props.params.StackColumnAt;
```

Comment:
Value of StackColumnAt property should be a breakpoint infix (e.g xs, sm, md). Sending a number value defeats all the benefits of customization and doesn't allow a front-end persona to control/change breakpoints from single place

Reply:
Updated!